src/modular/boxed_monty_form.rs

```diff
@@ -307,6 +319,14 @@ impl Monty for BoxedMontyForm {
     }
 }
 
+#[cfg(feature = "zeroize")]
+/// NOTE: This zeroizes the value, but _not_ the associated parameters!
```

Comment:
Following `std` conventions:

```suggestion
/// NOTE: This zeroizes the value, but _not_ the associated parameters!
#[cfg(feature = "zeroize")]
```